src/features/analysis/AccessibilityAnalysisForm.tsx

```diff
@@ -237,9 +247,8 @@ const AccessibilityAnalysisForm = (props: Props) => {
             value={form.formData.maxSlope}
             onChange={form.handleInputChange}
             type="number"
-            className=""
           />
-
+          <br />
```

Comment:
Is the br required or it was just cosmetic to have a blank line ?

Reply:
Both :)